api/overview/azure/storage.md

```diff
@@ -25,7 +25,6 @@ Use the following version 12.x libraries to work with blobs, files, and queues:
 | Library | Reference | Package | Source |
 |----------------------------------------|-------------------------------------------------------------|-----------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
 |    Azure.Storage.Blobs.Batch    |         |    [NuGet](https://www.nuget.org/packages/Azure.Storage.Blobs.Batch/)    |    [GitHub](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Blobs.Batch)    |
```

Comment:
Should we replace `master` with `main` to avoid the redirect?

Reply:
Good catch

Reply:
done changed it.